dbld/images/helpers/dependencies.sh

```diff
@@ -130,8 +130,7 @@ function install_pip_packages {
 
 function enable_dbgsyms {
     echo "deb http://ddebs.ubuntu.com $(lsb_release -cs) main restricted universe multiverse
-deb http://ddebs.ubuntu.com $(lsb_release -cs)-updates main restricted universe multiverse
```

Comment:
OPTIONAL: I prefer readability over reduction in case of built in scripts. e.g.: each addition in different lines, so they can be understand quickly, and modification do not affects multiple lines.
```
echo "deb 1.." | tee
echo "deb 2.." | tee
echo "deb 3.." | tee
etc...

```

Reply:
Thanks for the note. I have fixed this.